.github/workflows/update_protos.yml

```diff
@@ -31,18 +31,19 @@ jobs:
           cmakeVersion: 3.25.2
           ninjaVersion: 1.11.1
       - uses: bufbuild/buf-setup-action@v1.9.0
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Was `github_token`, the below `repo-token`, and the below `BUF_TOKEN` already unnecessary? Or made unnecessary by what you're doing here?

Reply:
I believe they've been unnecessary for a bit, we just haven't removed it